### DISCUSSION TITLE
Unreal Engine 5

### DISCUSSION THREAD

**Discussion by Francoporto** [chromiecraft/chromiecraft#7344] [Suggestions]
Yes indeed it is not a simple suggestion it is a real project, but I can not help but think that Turtle WoW is doing it from a client of 2004. Warmane only thinks about money with the bare minimum effort possible, no one else thinks, has the time and / or the passion to do this except Turtle WoW.

If having the script in open source today is something incredible, we are already in 2024 and the population of private servers has fallen for 10 years, it is no longer the golden age of WoW for a long time and especially on private.

We notice that in the general sense players want novelty, improvement and no longer loyalty to the old game.

Many people do not want to play anymore and the more reasons there are for this the less they want to resume the adventure consequently.
In other words, if we give players reasons, we increase the chances that they will come back.

I sincerely think that graphically we need something new. One of the first things that made us fans of the game was the wow effect, "wah this game is so beautiful, wah look at that so badass". The graphics contribute to the player's immersion and an immersed player is a happy player, on the contrary it is only an arcade game, a raider caught in its loop that only has interest in its dps and spamming of keybind in raid.

Adventure and discovery no longer exist in World of Warcraft and it is normal because the game no longer encourages us to do so.

We also note that the ergonomics of the client is old, the addons are not as incredible as retail, the optimization every 3 years takes a hit with the new hardware and the game is simply ugly, and the client does not allow the modder to improve it because they are limited by the API and the client.

Unreal Engine also allows for the integration of VR, DLSS, FSR and many other things.

I am writing this post knowing full well that this suggestion will never see the light of day because I already know the answers "It's too much work" "You can't imagine what it represents", but if nobody do that private server will just die in a couple of years and the whole script you are making will be just useless.

Turtle WoW are doing it so why not ?

**Comment by Helias** [comment]
AFAIK there were not many tentative of making a new WoW client from scratch, except for:
- [pseuwow](https://github.com/fgenesis/pseuwow) very old abandoned client for MaNGOS
- [novusengine](https://github.com/novusengine/Game) client, but I do not know if it's compatible with mangos-based core (like AzerothCore or TrinityCore)
- [Wowser](https://github.com/wowserhq/wowser) wow web client 
- and then the TurtleWoW client

The Turtle Wow client seems promising from the trailer, but the job is very hard, I do not know the status of it and how much is "complete", I hope the best for them and I think that they are very professional, indeed from the trailer seems that they built a complete version of the WoW-client.

On our side, we can't right now put the effort into this, unfortunately, but in the past, we did an [acore-client](https://github.com/azerothcore/acore-client) based on Wowser, it's more a copy I would say, and we integrated it into a launcher, this is our contribution that we take into the wow-client development landscape.

To build a new WoW client from scratch we should have an entire army of developers to build a new client from scratch, moreover, we need game client developers and not game-server developers, so a different category of developers.

About the wow-client development landscape I would like to find more time to improve the acore-client and at least make a "portable" client for WoW players that allows them to have a chat (in the browser and/or on mobile) which does not require a lot of effort thanks to wowser but not sure how much is requested from the players, we have several priorities than a wow client :sweat: 
For the Azerothcore/developers community, would be great to have it also for improving the testing, having a minimal working client could help a lot on testing, right now sometimes I need to open 6 WoW to test some 3v3 arena scenarios or other kinds of scenarios, a minimal wow client could make easier and faster the tests, but still, we really need an army for it.

These are my thoughts about developing an open-source wow client.

**Comment by Nyeriah** [comment]
Thanks for your suggestion however creating a new client is not within the scope of our project, so we will be rejecting this.
You could try opening a discussion at AzerothCore's organization instead to see if there's traction/interest from the wider community.
